network-api/networkapi/wagtailpages/templates/buyersguide/product_page.html

```diff
@@ -160,19 +160,19 @@ <h3 class="pni-product-smaller-body-bold mb-1 mb-md-2">{% trans "What data does
 
 
           {% trans "How does it use this data?" as how_does_it_use_data_collected %}
-          {% include "fragments/product_criterion.html"  with  label=how_does_it_use_data_collected  value=product.how_does_it_use_data_collected  ding=product.data_collection_policy_is_bad %}
+          {% include "fragments/product_criterion.html"  with  label=how_does_it_use_data_collected  value=product.how_does_it_use_data_collected  ding=product.data_collection_policy_is_bad rich_text_value=True %}
```

Comment:
while this will work, let's file a follow-up issue to see if can determine this "from the value itself" rather than assuming it's `True`, and then we can make that something the template can take care of for us, rather than needing us to hardcode it.